index.js

```diff
@@ -74,6 +53,16 @@ class BabelCompiler {
     this.options.filename = file.path;
     this.options.sourceFileName = file.path;
 
+    if (!this.options.presets) {
+      const babelConfig = new babel.OptionManager().init(this.options);
+      console.log(babelConfig);
+      const hasConfig = babelConfig.presets && babelConfig.presets.length > 0;
+
+      if (!hasConfig) {
+        this.options.presets = ['latest'];
```

Comment:
@denysdovhan I see 2 reasons:
1. I think we could add `targets` field to babel-brunch options. So, you can just specify targets in brunch-config and then brunch will check `preset-env` + `uglify` and set `uglify: true` for clean uglifying.
2. [browserslist config/field in package.json](https://github.com/babel/babel-preset-env/pull/161) will be released soon. Then env will look up `browserslist` configuration unless you specify `browserslist: false`. So to use babel-brunch with specific envs you could just add it to package.json and create browserslist config which can also be used for autoprefixer, eslint-plugin-comat, etc.